src/fields/FileField.h

```diff
@@ -0,0 +1,31 @@
+#ifndef FILEFIELD_H
+#define FILEFIELD_H
+
+#include "Field.h"
+#include <iostream>
+#include <fstream>
+#include <stdio.h>
+
+using namespace std;
+
+class FileField: public Field {
+
+    private:
+        string _fieldValue;
+        string _defaultValue;
+        string _folderPath;
+
+    public:
+        FileField(string fieldName, string folderPath="", string defaultValue="", bool isPrimaryKey=false, bool isUnique=false, bool isNullable=true, bool isAutoFilled=false);
+
+        void setValue(string value);
+        void saveNewFile(string pathToFile, string newFileName);
```

Comment:
Instead of `saveNewFile` just have a private `saveFile` method, as mentioned above.